src/inc/corcompile.h

```diff
@@ -71,7 +71,8 @@ typedef DPTR(struct CORCOMPILE_VIRTUAL_SECTION_INFO)
 typedef DPTR(struct CORCOMPILE_IMPORT_SECTION)
     PTR_CORCOMPILE_IMPORT_SECTION;
 
-#if defined(_TARGET_X86_) && !defined(FEATURE_PAL)
+#ifdef _TARGET_X86_
+#ifndef FEATURE_PAL
 //
 // x86 ABI does not define RUNTIME_FUNCTION. Define our own to allow unification between x86 and other platforms.
```

Comment:
We are not converged to the three field version even on non-x86 platforms today:

The only platform that uses the three field version is amd64 (on both Windows and Unix). 

arm and arm64 (on both Windows and Unix) use the two field version. Maybe that actually get compiled with the three field version on Unix, but the third field is just a dead weight. It is not used for anything. We should get rid of it.